test/DotNetEnv.Tests/EnvTests.cs

```diff
@@ -177,7 +177,9 @@ public void ParseInterpolatedTest()
             Assert.Equal("test:testtest1 $$ '\" ® and test1", Environment.GetEnvironmentVariable("TEST5_DOUBLE"));
             Assert.Equal("$TEST:$TEST4 \\$\\$ \" \\uae and $TEST1", Environment.GetEnvironmentVariable("TEST5_SINGLE"));
             Assert.Equal("test:testtest1\\uaeandtest1", Environment.GetEnvironmentVariable("TEST5_UNQUOTED"));
-
+            
+            Assert.Equal(" surrounded by spaces ", Environment.GetEnvironmentVariable("TEST_UNQUOTED_WITH_INTERPOLATED_SURROUNDING_SPACES"));
```

Comment:
added this test to prove, that leading whitespaces are now present as we expect them to be when we interpolate within an unquoted value.
The Trimming was throwing them away before, so we get the expected output now compared to before.

Reply:
Oh that is good